plugins/wls.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+    name: wls
+spec:
+    version: v1.0.0
+    homepage: 'https://github.com/mohatb/kubectl-wls'
+    shortDescription: 'Windows-linux-shell allows you to start a shell on windows or linux nodes'
+    description: "kubectl-wls stands for kubectl Kubectl-Windows-Linux-Shell, it is an interactive shell script and also a kubectl plugin that allows you to access a kubernetes windows or linux nodes shell.\n"
+    platforms:
+        -
+            selector: {matchExpressions: [{key: os, operator: In, values: [darwin, linux]}]}
+            uri: 'https://github.com/mohatb/kubectl-wls/archive/v1.0.0.tar.gz'
+            sha256: 1766ed7bb18dcf38cfe0a136246d731d94776b7292b632a668fba7bfb21b3f22
+            files: 
+              - from: '*'
+                to: "."
```

Comment:
this is optional, you can omit if you're copying out everything

Reply:
updated the yaml with the required files.